Tests/run_tests.py

```diff
@@ -161,6 +157,7 @@ def is_pypy():
 # Can't easily write exceptions with consistent class name in python 2 and 3
 if sys.version_info[0] == 2:
     EXCLUDE_DOCTEST_MODULES.append("Bio.Seq")
+    EXCLUDE_DOCTEST_MODULES.append("Bio.Phylo")
```

Comment:
Do we still need to exclude Python 2?

Reply:
Yes, this has failed on Python2.7 and pypy here:
https://travis-ci.org/biopython/biopython/jobs/544027693
https://travis-ci.org/biopython/biopython/jobs/544027698
because the values get rounded.